specification/peering/resource-manager/Microsoft.Peering/preview/2019-09-01-preview/peering.json

```diff
@@ -1871,9 +2400,12 @@
           "description": "The Autonomous System Number (ASN) of the peer.",
           "type": "integer"
         },
-        "peerContactInfo": {
-          "$ref": "#/definitions/ContactInfo",
-          "description": "The contact information of the peer."
+        "peerContactDetail": {
+          "description": "The contact details of the peer.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContactDetail"
+          }
         },
         "peerName": {
```

Comment:
adding new property is a breaking change.. this is also removing.. please limit the these to new api version